packages/thriver-core/lib/client/alerts.js

```diff
@@ -0,0 +1,21 @@
+Thriver.alert = {
+  /**
+   * @summary Display Alerts
+   * @method
+   */
+  hideAlert: () => {
+    // Vars
+    const alert = document.getElementById('alert');
+    alert.classList.remove('active');
+  },
+
+  showAlert: (opts) => {
+    // Define Alert
+    const alert = document.getElementById('alert');
+
+    // Update alert content & display alert
+    $(alert).find('.content').html(opts.content);
```

Comment:
We should not update html here, since that could potentially open us up for a XSS attack.  We should used textContent instead.